### DISCUSSION TITLE
helm uninstall my-release 报：Error: uninstall: Release not loaded: my-release: release: not found

### DISCUSSION THREAD

**Discussion by jhyever** [milvus-io/milvus#38368] [Announcement]
<img width="461" alt="f92e78e2965242608bc69683fc4f3ed" src="https://github.com/user-attachments/assets/3362d4d0-def2-40a4-a1fe-1a78a82b2941">
<img width="726" alt="230a2686abc9e2be6113a8bbe5298e3" src="https://github.com/user-attachments/assets/c8ddbc6d-8f86-4022-9f45-a6d6712e1245">
<img width="461" alt="f92e78e2965242608bc69683fc4f3ed" src="https://github.com/user-attachments/assets/e44952dd-dc03-4380-ab6e-067f03aed6f3">
1、镜像有但是提示还要拉
2、离线安装卸载不了

**Comment by LoveEachDay** [comment]
@jhyever Offline installation uses kubernetes manifests generated by `helm template`. So you cannot uninstall with `helm uninstall`.

You'd use `kubectl delete -f <offline-manifests>` instead to delete all the resources.
